### PR TITLE
Review discussion and conclusions

### PR DESCRIPTION
**Pull request by fbarranco**
Please, do pay special attention to the simplification of the last paragraph in the discussion.

### CODE REVIEW

File: edubots_surveys.Rnw [hunk 1]
```diff
@@ -345,22 +345,21 @@ role of MIM platforms in higher education is still a challenge, and why the opin
...
+\cite{yang2019opportunities}. Nevertheless, recent studies have shown their
 positive impact on students' academic performance \cite{perez2020rediscovering}
-and engagement \cite{differ.chat,abbas2021onlinechat} have led to a growing
+and engagement \cite{differ.chat,abbas2021onlinechat}, leading to a growing
 interest in using this technology in the (possibly virtual) classroom. Indeed,
-the use of chatbots in collecting course feedback from students in higher
-education improved students' response quality and boosted their enjoyment levels
-\cite{abbas2021Surveys}.  According to \cite{Roblyer2010}, using either tools
-such as mobile devices or teaching strategies based on gamification
+the using chatbots to collect course feedback from students in higher
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: Indeed, using chatbots

**Review by aguillenATC — CHANGES_REQUESTED** [review]

File: edubots_surveys.Rnw [hunk 2]
```diff
@@ -361,31 +361,31 @@ in \cite{Pimmer2019} adopted a quasi-experimental, survey-based approach to
...
-Several higher education chatbots' evaluation studies have been undertaken. For
-instance, a recent evaluation review study presented by Smutny and Schreiberova
-\cite{Smutny2020} examined 47 educational chatbots implemented in Facebook
-Messenger with the focus to identify characteristics and quality metrics such as
-language, subject matter and platform\textcolor{black}{, whereas the study undertaken by Pérez and collaborators
-\cite{perez2020rediscovering} aimed to categorize educational chatbots, according
-to their, purpose into service-oriented and teaching-oriented}. The first category
-includes those that provide service support such as the chatbot Ask Holly
-\cite{meetholly} and Dina \cite{santoso2018dinus}; both chatbots respond to
+Several studies have been conducted to evaluate chatbots in higher education. For
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: Several studies to evaluate chatbots in higher education have been conducted.

File: edubots_surveys.Rnw [hunk 3]
```diff
@@ -438,30 +438,29 @@ to develop effective chatbots for higher education, the needs of all stakeholder
...
 students in a Chinese class \cite{doi:10.1177/0735633120929622}. This paper
-analysed conversational chatbots in an one-to-one setting, finding that learning
-really benefited from it. This is, once again, a proof of the benefit of
-chatbots in certain settings; however, there are some prior experiences, as well as needs, that might
+analysed conversational chatbots in a one-to-one setting, finding that learning
+truly benefited from it. This is, once again, an evidence of the benefit of
+chatbots in certain settings; however, there are some prior experiences, as well as needs, that may
 prevent the successful acceptance of the technology, and these are what we are
-trying to find out in this paper, together with what kind of features would improve
+trying to find out in this paper, alng with what kind of features would improve
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: along

File: edubots_surveys.Rnw [hunk 4]
```diff
@@ -1438,33 +1438,33 @@ first was to check whether teachers already used some kind of messaging
...
+systems vs. web-based messaging systems integrated in another learning management system).
 
 This inclusion of university-provided messaging applications in
-the survey might explain why there are so few, just 6.7\%, that answer they do not
+the survey might explain why there are so few teachers, just 6.7\%, that answered they did not
 use any messaging application. In general, universities provide official channels
 to communicate with students official academic information such as the exam schedule
 or the grades. The statistical mode is to use that kind of messaging. Some other
-people might not use it, but use some extraofficial or opt-in channels such as the other included in the survey; at the end, between these official and extraofficial channels most people cover their needs to communicate with students. This matches
+teachers might not use it, but use some extraofficial or opt-in channels such as the others included in the survey; in the end, between these official and extra-official channels, most people cover their needs to communicate with students. This matches
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: teachers -> teaching staff

File: edubots_surveys.Rnw [hunk 5]
```diff
@@ -1468,35 +1468,31 @@ sciences show some significant differences. We do not have any kind of explanati
...
-experienced. This will probably require further study, but also will need to be
-taken into account when introducing chatbot technologies in the classroom:
-it will have to be as simple, and as satisfactory, as possible,
-to (possibly) change this existing perception.
+only listed a few functionalities for the chatbots, and just a few subjects selected
+the option "Others". This may suggest that most teachers did not really understand
+at this point the actual potential of chatbots, since the responses listed were by no means exhaustive. If the responders were more acquainted with the possibilities of this technology, this response would possibly have shown up many more times. Another way to look at it is that, unless you provide the specific instructions on how to use a
+tool, the teaching staff is already too busy or burnt out to think about innovation.
+
+We can speculate that teachers' opinions of chatbots are influenced by negative
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: teachers' -> teaching staff's

File: edubots_surveys.Rnw [hunk 6]
```diff
@@ -1468,35 +1468,31 @@ sciences show some significant differences. We do not have any kind of explanati
...
+experiences with customer support chatbots, which is what most people are likely to have
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: experiences with chatbots used in customer support,

File: edubots_surveys.Rnw [hunk 7]
```diff
@@ -1468,35 +1468,31 @@ sciences show some significant differences. We do not have any kind of explanati
...
+experienced. Further study may be needed to confirm this, but it is important to 
+keep in mind when introducing chatbot technologies in the classroom.
+To potentially change this existing perception, it will be necessary to make the chatbot as simple and satisfying to use as possible.
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: to make chatbots as simple

File: edubots_surveys.Rnw [hunk 8]
```diff
@@ -1468,35 +1468,31 @@ sciences show some significant differences. We do not have any kind of explanati
...
 
 Regarding gender grouping, both genders agree in their uses, so it seems not
-relevant at all. When looking at the plot containing the sector differentiation
-it is interesting to observe how university teachers prefer in a higher
-proportion to use them for grades and FAQs in comparison with other tertiary education teachers.
+relevant at all. However, when we look at the plot containing the sector differentiation,
+it is interesting to note that university teachers are more likely to use chatbots
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: university teaching staff

File: edubots_surveys.Rnw [hunk 9]
```diff
@@ -1468,35 +1468,31 @@ sciences show some significant differences. We do not have any kind of explanati
...
+keep in mind when introducing chatbot technologies in the classroom.
+To potentially change this existing perception, it will be necessary to make the chatbot as simple and satisfying to use as possible.
 
 Regarding gender grouping, both genders agree in their uses, so it seems not
-relevant at all. When looking at the plot containing the sector differentiation
-it is interesting to observe how university teachers prefer in a higher
-proportion to use them for grades and FAQs in comparison with other tertiary education teachers.
+relevant at all. However, when we look at the plot containing the sector differentiation,
+it is interesting to note that university teachers are more likely to use chatbots 
+for grades and FAQs compared to other tertiary education teachers.
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: education staff (o teaching staff)

File: edubots_surveys.Rnw [hunk 10]
```diff
@@ -1468,35 +1468,31 @@ sciences show some significant differences. We do not have any kind of explanati
...
 is lower at the University so, an automatic tool for grade notification for
-continuous evaluation seems more helpful.
-Furthermore, the high student/teacher
-ratios that university teachers have to deal with could be another reason to
-have a higher number of them willing to use chatbots for FAQs. This could
-ameliorate the number of misunderstandings and email overload that so many
-students might generate.
+continuous evaluation would be more helpful.
+Furthermore, the high student-to-teacher
+ratios that university teachers often face could be another reason why
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: ratios that university teaching staff

File: edubots_surveys.Rnw [hunk 11]
```diff
@@ -1494,23 +1494,22 @@ ratios that university teachers often face could be another reason why
...
 reduce misunderstandings and email overload from students.
 
-The results, when grouping by experience, confirm that
-experienced teachers are not afraid of changes and are willing to accept new
-technological challenges; at least, in the same way that younger teachers
-are. From this concrete plot it is interesting as well to observe the fact that very few
-young/inexperience teachers are willing to use chatbots for other uses. This
+The results, when grouping by teaching experience, confirm that
+experienced teachers are just as open to change and willing to embrace new
+technological challenges as younger teachers.
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: teachers -> educators

File: edubots_surveys.Rnw [hunk 12]
```diff
@@ -1513,21 +1513,20 @@ and synchrony as valuable in messaging apps.
...
- \item Teachers want to have support from their corresponding institution. This
-   is probably due to the strict European data regulations \textcolor{black}{on data protection},
-   but also to avoid the overhead required to \textcolor{black}{sign up new students every year.
-   In general support from specific formation and access to IT help desk for specific features
-   enhances the possibility of adoption by the teachers or the possibility of a successful
-   adoption that improves the learning outcomes. This is supported by the result of the two
-   surveys: they prefer whatever "messaging" application is provided by the university, and
-   also they value sustainability as well as "officiality" of the platform, as shown in
-   Table \ref{tab:interaction}}.
+ \item Teachers want to have support from their institutions. This
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: Hay que evitar el uso de teachers, creo que fue una revisión previa...

File: edubots_surveys.Rnw [hunk 13]
```diff
@@ -1513,21 +1513,20 @@ and synchrony as valuable in messaging apps.
...
+   is probably due to the strict European data regulations on data protection,
+   but also to avoid the overhead required to sign up new students every year.
+   In general, additional support with specific formation and access to IT help desk for specific features
+   enhance the possibility of a successful adoption that improves the learning outcomes. This is supported by the result of the two
+   surveys: they prefer whatever "messaging" application is provided by their university, and
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: whatever "messaging" application is provided by  -> whatever "messaging" application provided by

File: edubots_surveys.Rnw [hunk 14]
```diff
@@ -1513,21 +1513,20 @@ and synchrony as valuable in messaging apps.
...
+   they the "officiality" of the platform, as shown in
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: as well as the fact that this application is the ``official'' for the university, as shown in

File: edubots_surveys.Rnw [hunk 15]
```diff
@@ -1513,21 +1513,20 @@ and synchrony as valuable in messaging apps.
...
+   Table \ref{tab:interaction}.
  \item Teachers need sustainability. The main reason is that implementing
    changes is expensive in terms of effort and prone to errors in the first
-   years of implementing new procedures, therefore, it is logical that if
-   chatbots or messaging platforms are to be introduced, they want this change
+   years. Therefore, it is logical that if
+   chatbots or messaging platforms are to be introduced, teachers want this change
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: evitar teachers

File: edubots_surveys.Rnw [hunk 16]
```diff
@@ -1513,21 +1513,20 @@ and synchrony as valuable in messaging apps.
...
    to be as permanent as possible.
- \item Keep it Simple. Media interaction requirements are simple and only a 2.4\% is requesting more features. This could be a sign of technological burn out produced by the previous and current courses where, due to the pandemic, the use of computer has increased significantly.
+ \item Keep it Simple. Media interaction requirements are simple and only 2.4\% of teachers are requesting more features. This could be a sign of technological burnt out produced by the previous pandemic courses, when the use of computers significantly increased.
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: teachers

File: edubots_surveys.Rnw [hunk 17]
```diff
@@ -1529,29 +1529,27 @@ From a qualitative point of view, these results show consensus in a few key aspe
...
-  Spain, every 10 years or so.}), or simply introduction of new degrees. Although
-changes driven by external forces do offer the chance of piloting new technologies, they do not
-seem to bring internal changes in attitudes in the teaching staff (which, after all, is
+In our survey, we could not find an answer about the right moment to introducing chatbot technologies. The response to questions regarding to change of
+behavior during and after the first stay-at-home stage of the pandemic do not suggest that a major (or minor, such as new higher education laws or the introduction of new degrees) crisis could be an opportunity to introduce new technologies. The reason being that it does not
+bring major changes in attitude 
+%(as a minor "crisis" we could mention any institutional event that brings about methodological changes, such as new higher education laws\footnote{We seem to have one of these, at least in Spain, every 10 years or so.}), 
+%or simply the introduction of new degrees. 
+Although changes driven by external forces do offer the chance of piloting new technologies, they do not
+seem to bring internal changes in teachers' attitudes (which, after all, is
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: teaching staff's

File: edubots_surveys.Rnw [hunk 18]
```diff
@@ -1529,29 +1529,27 @@ From a qualitative point of view, these results show consensus in a few key aspe
...
-An analysis of survey questions related to RQ4 seems to indicate that teachers
-prefer to share an interaction space with their student, as
-shown in Tables \ref{tab:kindinteractions} and \ref{tab:interaction}, where a majority of responders
-indicate they {\em don't} want unidirectional communication; a preference of
-{\em horizontal} interaction, but also the answer to the kind of
-chatbots preferred that features FAQ bots prominently, which might indicate a
-need to delegate or offload part
-of the burden of answering, and attending, every single question posed by
-students. In an horizontal setting, other students can answer if the teacher
+The analysis of the questions related to RQ4 seems to indicate that teachers
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: that teaching staff

File: edubots_surveys.Rnw [hunk 19]
```diff
@@ -1586,31 +1586,26 @@ with higher retention in higher education \cite{o2014front}.  Therefore, both
...
-in students or in groups of them. These should also be accompanied by
-analytics on student interaction, as well as possibly some natural language
-processing (in vernacular language) that will help assess the general mood of
-the class, and how different material (or external factors) affect it.
+some easy automation or connection to personal instant messaging tools could
+help introduce these new technologies to the learning community.
+
+Based on the discussion of this study, we propose a potential process for introducing
+technology that would need to be piloted in order to determine its value and its
+relationship to improved learning outcomes and higher teacher satisfaction.
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: teacher -> educator/teaching staff

File: edubots_surveys.Rnw [hunk 20]
```diff
@@ -1608,32 +1608,32 @@ processing (in vernacular language) to help assess the general mood of
...
 the messaging applications used by tertiary education teachers, opens a new line
-of inquiry about what they {\em perceive} as such, and about how it is used.
-Namely, the responses indicated that the teachers perceived as messaging
-application not only what is usually called a chat or instant messaging app such
-as WhatsApp or Microsoft Teams, but also the means provided by the university for
-communicating with the student, such as a feature of grading applications that
-will email the grade to the student. This implies that there is a need by
-teachers, communication with students, also mostly unidirectional, and that it
-does not matter so much how that need is covered. However, this will need careful
+of inquiry about what teachers {\em perceive} as such, and how it is used.
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: educators

File: edubots_surveys.Rnw [hunk 21]
```diff
@@ -1635,23 +1635,23 @@ does not matter as much how that need is met. However, this will need careful
...
 teaching solutions that the COVID pandemic has brought has also taught us a
-series of lessons. It increases isolation, for instance, and decreases the
-amount of synchronous contact that happens in the fringes of the classroom:
-teaching staff offices, before and after class. A future line of work would be
-focused on exactly this, and what kind of needs could be covered by chatbot
-technology. We will create a series of international surveys that will
-investigate this.
-
-Finally, the full extent of chatbot technology is not really being examined in
+series of lessons: it increases isolation, for instance, and decreases the
```
Comment (aguillenATC) [inline edubots_surveys.Rnw]: remove for instance

**Review by aguillenATC — DISMISSED** [review]

File: edubots_surveys.Rnw (hunk 1 above)
Comment (fbarranco) [reply edubots_surveys.Rnw]: @aguillenATC, do it directly on the text. Just scroll over and click on the line when the blue cross appears. Next, click on Add a Suggestion (Ctrl+g) and change text with your suggestion (changes will be incorporated when accepted, automatically)

File: edubots_surveys.Rnw [hunk 22]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-Messenger with the focus to identify characteristics and quality metrics such as
-language, subject matter and platform\textcolor{black}{, whereas the study undertaken by Pérez and collaborators
-\cite{perez2020rediscovering} aimed to categorize educational chatbots, according
-to their, purpose into service-oriented and teaching-oriented}. The first category
-includes those that provide service support such as the chatbot Ask Holly
-\cite{meetholly} and Dina \cite{santoso2018dinus}; both chatbots respond to
+students' knowledge and mitigating their feelings of isolation.
+
+Several studies have been conducted to evaluate chatbots in higher education. For
+instance, a recent study was presented by Smutny and Schreiberova
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
instance, a comprehensive study was presented by Smutny and Schreiberova
```

File: edubots_surveys.Rnw [hunk 23]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+\cite{Smutny2020} reviewing 47 educational chatbots implemented in Facebook
+Messenger, with a focus on identifying characteristics and quality metrics such as
+language, subject matter, and platform. Pérez and collaborators
```
Comment (JJ) [inline edubots_surveys.Rnw:366]: This is probably outside the scope of the revision, but I feel we might need to clarify this. Characteristics of what? What was the conclusion?
Alternatively, just skim the paper that make a more focused comment on it, mainly on its conclusions.

File: edubots_surveys.Rnw [hunk 24]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+\cite{perez2020rediscovering} attempted to categorize educational chatbots attending to their purpose 
+as service-oriented or teaching-oriented. Service-oriented chatbots
+include those that provide service support, such as Ask Holly
+\cite{meetholly} and Dina \cite{santoso2018dinus}, which responds to
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
\cite{meetholly} and Dina \cite{santoso2018dinus}, which respond to
```

File: edubots_surveys.Rnw [hunk 25]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-proposed a generic chatbot for university students that is able to answer to
-their frequently asked questions \textcolor{black}{and, for some time, the University of Granada had
-\textcolor{black}{Elvira} as a chatbot in its main web page \cite{MOREO20129} to perform that duty.} Besides being able
-to answer pre-established (frequent) questions, it did so from the website of
-the University of Granada using an inset \textcolor{black}{persona, voiced by a real person, who
-lip-synced the answers. The emphasis in the case of Elvira, however, was on the
-authenticity of the synchronization more than in making the update of questions
-answered simpler, or more interactive, or more open to the rest of the university staff, using actual messaging platforms. In
+proposed a generic chatbot for university students that is able to answer 
+frequently asked questions. The University of Granada had
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
frequently asked questions. The University of Granada deployed
```

File: edubots_surveys.Rnw [hunk 26]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+Elvira, a built-in chatbot in its main web page \cite{MOREO20129} to perform the same task. In addition to being able
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
Elvira, a chatbot embedded in its main web page \cite{MOREO20129} to perform the same task. In addition to being able
```

File: edubots_surveys.Rnw [hunk 27]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+to answer pre-established frequent questions, it did so from the website of
+the University of Granada using an inset persona, voiced by a real person, who
+lip-synced the answers. However, in the case of Elvira, the emphasis was on the
+authenticity of the synchronization rather than making the updated
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
authenticity of the speech-mouth and face gestures synchronization rather than making updating
```

File: edubots_surveys.Rnw [hunk 28]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-authenticity of the synchronization more than in making the update of questions
-answered simpler, or more interactive, or more open to the rest of the university staff, using actual messaging platforms. In
+proposed a generic chatbot for university students that is able to answer 
+frequently asked questions. The University of Granada had
+Elvira, a built-in chatbot in its main web page \cite{MOREO20129} to perform the same task. In addition to being able
+to answer pre-established frequent questions, it did so from the website of
+the University of Granada using an inset persona, voiced by a real person, who
+lip-synced the answers. However, in the case of Elvira, the emphasis was on the
+authenticity of the synchronization rather than making the updated 
+of answered questions simpler, more interactive, or more open to the rest of the university staff using actual messaging platforms. In
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
answered questions simpler or more interactive for the administrative staff, or more open to the rest of the university staff using actual messaging platforms. In
```

File: edubots_surveys.Rnw [hunk 29]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
 create interactive instances in their lectures where they pose questions to
 their students and the chatbot assists those who need further help by giving
-them more hints and reporting it to their educator's dashboard
-\cite{fernoagua2018intelligent}. This can help educators locate those
-students that need more attention and send them more educational resources
-relevant to their academic attainment. \textcolor{black}{In addition, a recent study
-has analysed how the use of chatbots positively affected the learning outcomes
-of students in a Chinese class \cite{doi:10.1177/0735633120929622}}.
+them more hints and reporting it to the educator's dashboard
+\cite{fernoagua2018intelligent}. This can help educators identify
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
\cite{fernoagua2018intelligent}. This could help educators identify
```

File: edubots_surveys.Rnw [hunk 30]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+students who need more attention and provide them with more educational resources
+relevant to their academic achievement. %In addition, a recent study
```
Comment (JJ) [inline edubots_surveys.Rnw:404]: Why did you comment this out?

File: edubots_surveys.Rnw [hunk 31]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-appropriate types of questions that the chatbots should support for different
-courses.  \textcolor{black}{Some other authors}, like \cite{coronado2018cognitive}, proposed agents that
-store learning materials to be provided on demand to students, whereas the
-authors in \cite{crockett2017predicting} reported on tutoring systems, which can
+with and can easily access (i.e. Facebook Messenger), which was confirmed by
+\cite{hobert2019you} and \cite{fernoagua2018intelligent}. In addition, \cite{sjostrom2018designing} argued that 
+conceptualizing learners' questions could help designers integrate the
+appropriate types of questions that chatbots should support for different
+courses. Other authors, such as \cite{coronado2018cognitive}, have proposed agents that
+store learning materials to be provided on demand to students, while \cite{crockett2017predicting}
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
record learning materials to be provided on demand to students, while \cite{crockett2017predicting} 
```

File: edubots_surveys.Rnw [hunk 32]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+\cite{sjostrom2018designing,tsivitanidouusers}. These needs include, but are not limited to, student learning success due to higher motivation, but these are a posteriori effects that cannot be assessed in advance. This is why we focus in this paper on what  teachers are looking for in a Technology Acceptance Model (TAM) and associated technologies like chatbots. Both authors focus on what teachers need in terms of the latter; this paper will focus on a wider perspective, trying to determine what they are looking for in terms of general messaging technology. The classical literature \cite{tam91} already proposes that any adoption of technology must be tuned to the user needs and experience. In this paper, we will try to find out what those are in order to propose a successful model of adoption of chatbot technology.
```
Comment (JJ) [inline edubots_surveys.Rnw]: I don't really get this:

> This is why we focus in this paper on what teachers are looking for in a Technology Acceptance Model (TAM) and associated technologies like chatbots.

Do we really use a Technology Acceptance Model? "We are looking for... associated technologies like chatbots"?
Probably delete that whole sentence.

```suggestion
\cite{sjostrom2018designing,tsivitanidouusers}. These needs include, but are not limited to, student learning success due to higher motivation, but these are a posteriori effects that cannot be assessed in advance. Both authors focus on what teachers need in terms of the latter; this paper will focus on a wider perspective, trying to determine what they are looking for in terms of general messaging technology. The classical literature \cite{tam91} already proposes that any adoption of technology must be tuned to the user needs and experience. In this paper, we will try to find out what those are in order to propose a successful model of adoption of chatbot technology.
```

File: edubots_surveys.Rnw [hunk 33]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
 
-Along this line of research, our previous work \cite{MoraChatbots2021:anon}
-aimed at analyzing the expectations of students on this regard; recent papers
+In line with this research, our previous work \cite{MoraChatbots2021:anon}
+aimed at analyzing the expectations of students in this regard. Recent papers
```
Comment (JJ) [inline edubots_surveys.Rnw:440]: Again, how recent?

File: edubots_surveys.Rnw [hunk 34]
```diff
@@ -1440,126 +1438,118 @@ first was to check whether teachers already used some kind of messaging
...
-company) as well as the {\em type} of technology (instant, app based messaging
-systems vs. web-based messaging systems integrated in another learning management system).}
+company) as well as the {\em type} of technology (instant, app-based messaging
+systems vs. web-based messaging systems integrated in another learning management system).
 
 This inclusion of university-provided messaging applications in
-the survey might explain why there are so few, just 6.7\%, that answer they do not
+the survey might explain why there are so few teachers, just 6.7\%, that answered they did not
 use any messaging application. In general, universities provide official channels
 to communicate with students official academic information such as the exam schedule
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
to send students academic information such as the exam schedule
```

File: edubots_surveys.Rnw [hunk 35]
```diff
@@ -1440,126 +1438,118 @@ first was to check whether teachers already used some kind of messaging
...
-  Spain, every 10 years or so.}), or simply introduction of new degrees. Although
-changes driven by external forces do offer the chance of piloting new technologies, they do not
-seem to bring internal changes in attitudes in the teaching staff (which, after all, is
+In our survey, we could not find an answer about the right moment to introducing chatbot technologies. The response to questions regarding to change of
+behavior during and after the first stay-at-home stage of the pandemic do not suggest that a major (or minor, such as new higher education laws or the introduction of new degrees) crisis could be an opportunity to introduce new technologies. The reason being that it does not
+bring major changes in attitude 
+%(as a minor "crisis" we could mention any institutional event that brings about methodological changes, such as new higher education laws\footnote{We seem to have one of these, at least in Spain, every 10 years or so.}), 
+%or simply the introduction of new degrees.
```
Comment (JJ) [inline edubots_surveys.Rnw:1543]: Why did you delete this?

**Review by JJ — CHANGES_REQUESTED** [review]
Main hurdle is justifying deletions; please consider the rest of the comments too.

File: edubots_surveys.Rnw (hunk 35 above)
Comment (fbarranco) [reply edubots_surveys.Rnw:1543]: I moved part of it to the brackets in line 1547. It is hard to read, because we mention major and minor crisis and then, when writing about the reason we explain again what a minor crisis is ... 
Sorry, I just realized I left the footnote out, please put it back (in line 1547)

Comment (JJ) [reply edubots_surveys.Rnw:1543]: Not clear what you mean here. Just modify this PR...

File: edubots_surveys.Rnw (hunk 33 above)
Comment (fbarranco) [reply edubots_surveys.Rnw:442]: It is already said in the previous paragraph

File: edubots_surveys.Rnw (hunk 30 above)
Comment (fbarranco) [reply edubots_surveys.Rnw:404]: It is already said in the previous paragraph (I think I explained it in the commit ...)

File: edubots_surveys.Rnw [hunk 36]
```diff
@@ -317,17 +317,17 @@ The widespread and rapid adoption of free Mobile Instant Messaging
 (MIM) tools/platforms such as WhatsApp, Telegram, WeChat and Facebook
 Messenger stems from their simplicity, ease of use and multi-modality
 (i.e. video, audio, text) \cite{tang2017mobile}. Using these tools in
-higher education can facilitate the delivery of personalized learning
-that occurs anytime anywhere, promote collaborative learning
+higher education facilitates the delivery of personalized learning
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
higher education institutions facilitates the delivery of personalized learning
```

File: edubots_surveys.Rnw (hunk 1 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
using chatbots to collect course feedback from students in higher
```

File: edubots_surveys.Rnw (hunk 2 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
Several studies to evaluate chatbots in higher education have been conducted. For
```

File: edubots_surveys.Rnw (hunk 4 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
teaching staff might not use it, but use some extraofficial or opt-in channels such as the others included in the survey; in the end, between these official and extra-official channels, most people cover their needs to communicate with students. This matches
```

File: edubots_surveys.Rnw (hunk 5 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
We can speculate that teaching staff's opinions of chatbots are influenced by negative
```

File: edubots_surveys.Rnw (hunk 6 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
experiences with chatbots used in customer support, which is what most people are likely to have
```

File: edubots_surveys.Rnw (hunk 17 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
seem to bring internal changes in educators' attitudes (which, after all, is
```

File: edubots_surveys.Rnw (hunk 18 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
The analysis of the questions related to RQ4 seems to indicate that teaching staff
```

File: edubots_surveys.Rnw (hunk 19 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
relationship to improved learning outcomes and higher teacher staff satisfaction.
```

File: edubots_surveys.Rnw (hunk 20 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
of inquiry about what educators {\em perceive} as such, and how it is used.
```

File: edubots_surveys.Rnw (hunk 21 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
series of lessons: it increases isolation and decreases the
```

File: edubots_surveys.Rnw (hunk 3 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
trying to find out in this paper, along with what kind of features would improve
```

File: edubots_surveys.Rnw (hunk 7 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
To potentially change this existing perception, it will be necessary to make chatbots as simple and satisfying to use as possible.
```

File: edubots_surveys.Rnw (hunk 8 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
it is interesting to note that university teaching staff are more likely to use chatbots 
```

File: edubots_surveys.Rnw (hunk 9 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
for grades and FAQs compared to other tertiary education teaching staff.
```

File: edubots_surveys.Rnw (hunk 10 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
ratios that university educators often face could be another reason why
```

File: edubots_surveys.Rnw (hunk 11 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
technological challenges as younger educators. 
```

File: edubots_surveys.Rnw (hunk 12 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
 \item Teaching staff want to have support from their institutions. This
```

File: edubots_surveys.Rnw (hunk 13 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
   surveys: they prefer whatever ``messaging'' application provided by
```

File: edubots_surveys.Rnw (hunk 14 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
  as well as the fact that this application is the ``official'' for the university, as shown in
```

File: edubots_surveys.Rnw (hunk 15 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
   chatbots or messaging platforms are to be introduced, teaching staff want this change
```

File: edubots_surveys.Rnw (hunk 16 above)
Comment (aguillenATC) [reply edubots_surveys.Rnw]:
```suggestion
 \item Keep it Simple. Media interaction requirements are simple and only 2.4\% of educators are requesting more features. This could be a sign of technological burnt out produced by the previous pandemic courses, when the use of computers significantly increased.
```

**Review by JJ — APPROVED** [review]
As long as we clarify what's "recent" and what's not, I think we're good with this. Thanks!
